libraries/RW/K8s/namespace_tasks_mixin.py

```diff
@@ -235,3 +242,116 @@ def get_event_count(
             return len(event_rows)
         else:
             return 0
+
+    def _convert_age_to_search_time (self, age) -> str: 
+        current_time = datetime.now()
+        age = int(age.split('m')[0])
```

Comment:
Minor nitpick: I try default `"` for string quotes, falling back to `'` when trying to improve readability because I'm getting creative in the string value.